example/pubspec.yaml

```diff
@@ -6,10 +6,10 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  flame: 0.24.0
+  flame: ^1.0.0-rc8
```

Comment:
Can you update this to `1.0.0-releasecandidate.13`?

Reply:
OK!